### PR TITLE
Implement library pymunk for physic

### PR DESCRIPTION
**Pull request by n2oneProgrammer**

### CODE REVIEW

**Review by jaanonim — APPROVED** [review]
LGTM

File: maps/test_map.map [hunk 1]
```diff
@@ -38,14 +72,19 @@
     },
     {
       "type": "sprite",
-      "asset": "player",
-      "position": [
-        50,
-        100
+      "asset": "wall3.2",
+      "stretch": "yes",
```
Comment (mwawrzen) [inline maps/test_map.map:76]: Is it possible or bad idea to use boolean values instead of strings?

File: tests/map_builder_test.py [hunk 2]
```diff
@@ -180,10 +180,11 @@ def test_type_rect_returning_object(self):
             width=30,
             height=20,
             position=Vector2(40, 10),
-            color="#123456"
+            color="#123456",
```
Comment (mwawrzen) [inline tests/map_builder_test.py:183]: Omg, where did you get this beautiful color ?? ^.^

**Review by mwawrzen — APPROVED** [review]

File: maps/test_map.map (hunk 1 above)
Comment (n2oneProgrammer) [reply maps/test_map.map:76]: It's not bad idea, but I'm lazy and I don't want to change it

**Pull request merged**